swagger.yaml

```
@@ -5302,6 +5567,10 @@ components:
           block:
             type: string
             description: Block hash of the UTXO
+          data_hash:
+            type: string
+            description: The hash of the transaction output datum
+            nullable: true
         required:
           - tx_hash
           - tx_index
```

```suggestion
        required:
          - data_hash
          - tx_hash
          - tx_index
```

`data_hash` is missing from required properties.

If possible, please add it to the end of the list, it's not possible via suggestion.